packages/forms/backend/sirius-components-collaborative-widget-reference/src/main/java/org/eclipse/sirius/components/collaborative/widget/reference/ReferenceWidgetDefaultCreateElementHandler.java

```diff
@@ -21,15 +21,13 @@
 import org.eclipse.sirius.components.core.api.IEditService;
 import org.eclipse.sirius.components.core.api.IEditingContext;
 import org.eclipse.sirius.components.widget.reference.IReferenceWidgetCreateElementHandler;
-import org.springframework.stereotype.Service;
 
 /**
  * * Default implementation of {@link IReferenceWidgetCreateElementHandler}.
  * * This implementation use {@link IEditService}.
  *
  * @author frouene
  */
-@Service
 public class ReferenceWidgetDefaultCreateElementHandler implements IReferenceWidgetCreateElementHandler {
```

Comment:
The `canHandle` should also return true to be consistent with `ReferenceWidgetDefaultCandidateSearchProvider`